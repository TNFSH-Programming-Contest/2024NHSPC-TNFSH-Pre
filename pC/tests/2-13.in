84 1569
82 50
26 72
64 75
81 56
14 60
13 16
46 81
19 82
83 47
47 42
3 69
15 75
35 81
61 27
69 14
54 37
60 67
47 70
47 5
60 50
24 4
9 15
41 49
70 53
24 81
25 11
46 63
26 47
14 48
5 36
30 47
50 54
53 63
23 27
14 77
84 38
26 4
46 32
84 51
71 62
36 27
26 34
66 57
17 60
32 28
37 57
2 36
69 35
53 48
81 59
39 54
71 52
56 69
42 52
35 40
16 56
50 4
28 63
18 36
7 1
64 21
44 39
54 10
51 8
44 57
80 26
3 9
64 34
66 12
40 55
69 71
54 43
37 2
48 75
6 84
30 83
23 16
46 59
52 16
31 79
72 28
17 79
57 24
61 17
48 38
79 4
14 7
1 13
81 6
56 13
6 76
43 83
72 24
45 81
75 51
58 33
7 23
74 58
42 34
10 32
46 40
6 68
10 4
42 48
75 9
26 2
40 32
73 76
11 57
39 9
44 8
52 3
19 62
40 25
38 83
81 52
54 6
63 82
19 14
26 74
40 73
2 83
55 20
21 30
8 63
55 48
28 24
41 26
55 73
40 76
28 15
42 37
39 80
44 62
74 17
36 19
48 33
21 24
4 56
2 19
4 21
29 27
72 12
36 43
37 64
62 55
1 43
2 1
29 83
70 73
35 72
8 12
12 19
1 60
82 37
13 4
8 56
46 76
76 79
83 78
44 6
19 71
66 31
49 83
76 27
11 79
62 70
44 19
42 3
37 8
63 24
11 29
34 80
69 40
72 77
47 29
61 23
70 45
76 67
24 62
20 49
70 51
55 34
2 80
19 42
38 37
83 48
28 46
81 4
77 35
7 27
35 34
68 59
76 74
39 55
50 49
2 55
64 26
57 78
25 42
8 26
62 51
23 39
19 76
32 30
58 32
10 83
78 41
74 10
30 37
9 64
77 12
11 6
62 42
63 40
45 49
73 24
11 9
33 44
58 9
17 62
18 48
56 61
80 1
67 17
26 81
55 65
56 47
47 19
71 65
4 7
19 24
24 84
34 45
5 9
9 8
49 76
77 18
7 5
50 17
51 35
44 67
19 77
44 29
13 51
6 56
9 37
18 33
23 10
2 81
33 76
56 59
21 61
1 23
3 40
34 44
41 73
65 73
6 14
31 27
48 9
53 18
6 36
72 81
36 28
30 31
56 80
26 7
20 13
84 32
27 39
49 65
38 25
38 51
5 4
43 63
39 68
18 78
46 39
30 15
69 79
51 10
73 42
68 77
29 74
11 43
39 76
13 9
59 27
25 84
38 43
27 70
16 83
75 25
34 73
67 30
63 1
75 14
69 24
81 77
50 25
79 30
54 22
81 66
65 76
23 83
42 57
76 50
68 2
82 9
79 78
12 1
16 75
44 7
41 2
59 37
74 54
67 63
2 27
56 44
9 63
1 45
71 25
80 50
66 24
60 84
39 66
47 77
76 22
37 19
59 28
34 66
57 26
60 72
17 9
78 11
78 33
31 42
75 33
40 53
34 23
66 21
68 8
26 50
70 55
72 31
46 72
80 51
7 20
27 11
58 64
1 11
23 28
43 73
54 2
23 46
33 16
2 22
26 82
49 11
11 46
25 60
21 41
84 20
69 60
62 59
16 49
79 26
49 27
43 18
9 52
74 38
66 75
15 3
29 33
71 47
34 32
10 54
53 21
35 67
53 67
75 44
9 47
78 48
57 60
61 53
50 24
25 13
42 65
61 70
65 37
44 15
72 34
77 10
32 76
82 75
27 50
83 39
32 77
16 29
84 3
60 36
16 67
9 61
64 50
22 53
35 59
11 41
6 5
18 4
27 80
28 31
74 9
35 23
11 28
3 22
17 26
31 32
57 71
84 83
49 24
64 43
77 45
15 41
1 32
83 8
44 84
74 55
8 10
10 40
70 64
54 14
7 79
69 58
20 26
19 13
30 22
75 35
7 31
77 71
41 1
10 17
75 58
82 4
79 20
42 51
79 32
43 77
42 70
15 59
35 55
5 2
82 59
17 49
47 68
49 29
40 14
59 43
84 75
4 68
17 38
15 84
72 78
82 41
31 78
74 40
20 45
84 52
15 63
36 54
4 36
56 57
42 60
47 51
78 47
27 82
9 70
22 47
20 67
74 36
10 43
74 25
32 64
10 70
5 6
61 52
32 70
39 58
52 48
15 47
19 61
21 56
74 3
30 38
54 46
80 17
71 50
80 12
37 6
79 44
27 26
1 67
63 31
79 46
74 13
57 31
16 45
61 60
63 65
28 52
20 61
58 27
19 6
55 7
49 15
38 65
75 10
54 66
11 58
29 84
41 10
10 56
17 68
22 23
36 67
69 5
66 83
76 37
39 69
84 71
65 6
54 15
27 51
76 18
36 41
35 79
71 30
16 74
37 21
3 1
60 45
61 6
2 48
75 67
62 47
5 57
67 37
14 78
20 82
49 9
74 57
28 35
17 13
20 39
3 35
12 51
70 77
5 30
44 26
74 41
79 64
26 66
46 38
55 46
24 46
25 76
29 34
43 17
8 41
41 19
31 81
79 56
79 48
65 24
30 63
8 64
82 21
11 69
21 3
16 7
44 69
18 47
32 60
61 41
57 51
29 12
11 7
45 2
9 73
66 70
38 10
1 53
79 19
26 25
76 34
65 46
2 61
20 80
57 20
31 30
30 2
82 1
6 55
46 78
52 19
52 66
82 74
46 19
16 39
12 30
77 1
66 35
70 60
84 67
62 73
57 38
62 38
3 81
37 70
69 34
3 80
45 38
21 65
58 83
68 72
29 48
47 44
74 56
62 82
20 8
61 64
80 10
9 24
3 53
18 61
73 43
19 12
17 31
16 47
12 70
22 56
30 9
23 30
69 32
13 47
44 11
33 14
48 10
69 70
4 35
12 16
33 43
2 64
50 77
4 46
84 77
2 21
44 81
49 82
22 51
83 74
24 14
81 34
31 68
2 82
40 45
33 9
30 16
29 4
74 47
54 45
2 63
8 48
44 4
43 44
68 1
26 78
19 68
82 3
52 53
5 1
31 71
31 23
54 56
19 56
76 21
49 57
38 80
47 59
64 47
39 52
59 77
69 48
71 28
15 74
1 6
44 25
74 84
15 52
19 31
28 84
58 37
42 7
77 83
34 18
36 70
59 32
26 68
15 12
45 11
27 24
63 35
18 1
71 43
82 61
76 45
39 42
16 58
13 23
83 75
55 12
50 67
70 1
12 22
54 80
54 41
80 69
59 78
67 38
16 11
38 27
35 24
9 38
4 84
62 79
24 3
69 23
42 58
20 38
46 9
51 45
27 44
49 75
60 22
39 56
59 9
78 36
41 45
76 24
17 73
62 6
24 68
5 14
82 81
66 14
14 13
71 7
27 30
31 21
75 53
30 76
64 12
37 17
37 32
9 83
30 33
31 43
2 29
6 53
34 14
30 40
73 28
50 73
65 17
32 12
9 1
84 68
20 78
30 74
78 19
64 41
32 49
21 5
67 64
76 13
43 53
27 14
72 80
26 71
62 65
10 63
7 52
3 16
70 67
62 43
28 2
16 20
64 30
60 3
73 26
43 7
47 2
6 25
26 76
28 39
53 78
72 37
69 26
66 67
61 66
2 4
2 39
18 3
77 78
15 36
28 21
13 12
27 78
17 66
68 46
60 53
13 36
33 19
17 2
4 11
33 70
35 50
76 58
59 52
2 51
35 16
38 6
28 18
6 82
36 77
32 24
66 4
56 40
40 64
74 19
34 36
26 67
32 79
77 69
33 41
49 77
42 29
14 43
67 40
52 6
4 73
23 19
45 5
78 67
34 49
9 7
54 29
16 63
14 63
18 60
17 53
59 42
82 77
40 38
84 69
45 8
65 2
5 63
22 34
47 55
4 59
41 12
68 52
71 8
48 67
35 56
37 66
15 25
22 48
53 28
14 38
42 32
81 50
79 83
69 7
10 64
74 68
79 41
63 73
45 52
20 24
67 20
37 14
66 60
55 33
58 68
7 70
12 28
64 5
52 38
24 34
70 29
57 15
37 41
60 19
17 6
3 28
72 40
50 38
6 45
65 82
10 76
8 60
67 74
8 4
6 2
12 39
59 45
17 45
49 43
43 55
60 48
62 13
45 62
82 60
55 32
60 33
60 74
26 38
53 4
57 9
37 51
10 78
79 49
59 41
51 64
79 50
33 24
68 34
42 46
43 58
19 70
49 16
25 9
64 19
44 9
56 70
29 40
46 25
63 23
57 32
35 49
74 32
24 5
16 53
72 52
62 48
68 54
53 69
52 58
29 39
71 10
59 76
30 14
26 6
8 50
63 42
72 29
31 33
16 26
18 50
22 59
3 79
34 15
2 10
6 57
56 7
13 3
16 57
52 47
31 8
42 30
62 52
2 37
11 74
12 48
66 29
21 34
36 49
14 51
22 18
30 18
28 26
38 63
72 65
50 32
4 48
5 26
23 56
3 59
54 84
47 18
56 30
39 20
47 38
41 24
18 82
72 13
67 68
62 33
14 20
29 50
61 67
47 82
4 64
20 43
58 8
26 19
14 67
19 34
37 73
3 38
84 31
76 9
63 12
45 39
55 29
61 58
56 2
18 25
22 4
30 48
5 79
52 27
19 20
57 12
55 76
36 55
2 17
57 83
49 53
74 12
52 72
38 68
81 53
50 31
64 27
23 9
23 5
4 47
67 39
36 80
54 49
78 74
25 52
68 33
66 45
84 10
55 83
51 79
2 50
47 49
83 4
8 39
43 32
66 5
66 47
1 26
26 27
72 42
38 31
8 76
34 57
70 71
62 31
83 71
28 14
11 53
15 21
81 73
67 24
16 59
81 5
64 15
21 9
14 50
8 70
11 22
72 11
49 48
11 23
67 49
83 19
20 75
41 53
9 77
68 7
50 45
30 69
31 74
1 52
19 11
15 80
40 37
24 38
37 49
26 13
64 77
63 11
58 34
73 80
41 47
16 43
56 27
1 10
15 68
9 56
13 66
66 44
62 76
19 67
84 36
2 16
12 49
14 23
77 61
80 53
81 22
22 57
39 21
66 7
15 14
40 83
53 72
58 21
33 60
59 26
4 54
82 53
62 58
22 26
7 72
24 58
5 49
55 71
40 18
58 2
2 15
58 44
17 8
28 51
36 52
74 72
48 58
62 30
5 54
34 61
43 21
21 46
82 11
51 29
79 66
29 79
25 41
81 74
42 8
37 10
30 32
6 42
38 21
31 51
35 43
7 42
2 53
27 72
41 3
29 52
7 73
44 55
13 75
64 78
67 57
58 13
13 2
20 40
77 44
45 55
65 14
65 43
31 59
8 57
6 39
71 33
62 72
8 47
47 46
54 7
28 16
30 70
19 17
7 34
77 76
39 24
66 6
59 50
50 39
5 65
15 16
66 36
55 5
75 59
78 25
62 50
17 28
1 37
79 68
6 15
8 67
16 51
36 20
69 72
68 63
56 75
16 79
12 11
55 37
55 59
65 60
74 79
39 62
54 76
19 69
68 13
80 70
15 7
73 31
46 13
83 28
29 69
48 21
79 47
41 31
76 38
22 71
57 62
71 15
64 71
82 30
7 6
66 63
71 54
17 48
71 29
31 77
80 35
64 35
74 5
70 68
13 64
65 26
47 67
14 25
45 84
36 3
52 35
27 17
18 42
58 66
15 48
64 60
28 29
59 74
64 55
1 61
46 84
35 27
16 32
5 33
24 11
3 63
53 55
11 52
53 14
64 80
84 58
35 13
52 57
61 8
43 40
37 80
25 64
75 70
38 66
47 20
53 45
12 26
49 68
57 1
79 84
79 25
48 74
17 55
10 11
46 50
52 17
58 7
28 62
39 32
1 8
30 49
51 9
61 38
4 30
3 8
37 36
6 33
26 52
48 34
80 61
42 61
43 50
33 63
10 20
60 41
58 19
59 66
82 8
78 7
39 65
52 20
42 38
12 84
84 73
6 72
33 83
48 54
23 64
47 76
15 17
25 59
30 5
30 34
49 18
10 22
7 36
20 76
45 21
10 53
46 37
23 51
37 29
41 51
56 67
5 18
57 2
24 31
71 32
43 69
34 3
74 49
45 32
48 23
63 58
17 5
42 1
25 24
80 77
36 8
67 73
43 30
65 83
50 7
60 26
4 9
15 53
24 43
48 56
62 15
50 65
58 80
8 75
37 46
37 18
9 80
39 75
54 70
19 48
50 13
75 46
21 62
73 52
52 34
44 50
38 49
51 58
5 3
76 42
76 7
81 19
81 1
7 48
18 56
25 17
78 37
14 68
40 15
3 68
21 51
81 42
60 75
33 65
36 63
82 22
33 46
65 23
35 44
66 84
82 16
18 80
70 57
47 43
56 73
63 16
18 64
67 2
82 48
51 77
77 67
3 73
36 46
31 49
33 22
76 28
60 31
2 79
23 53
6 63
79 82
70 16
33 51
26 18
26 23
68 53
75 32
44 40
13 63
75 62
43 51
62 57
73 74
32 5
23 49
49 21
59 38
71 11
47 23
36 58
49 84
52 43
43 6
29 13
24 77
30 8
31 3
60 10
54 47
14 16
28 77
11 50
25 35
72 8
61 62
50 23
84 61
14 41
59 13
25 40
49 26
2 75
31 22
43 67
6 58
46 74
78 62
44 22
68 10
64 7
79 72
28 49
63 17
45 9
75 4
71 42
16 4
41 62
44 80
52 41
82 72
28 44
33 34
40 9
50 75
67 42
73 66
32 7
49 1
30 25
10 46
26 54
73 75
30 80
45 23
13 71
32 35
39 77
49 8
16 55
50 20
7 9
66 16
12 31
4 76
76 53
43 2
73 82
77 40
51 66
40 8
36 76
77 6
67 22
40 75
23 57
84 39
49 19
54 17
49 61
59 57
46 27
21 43
62 26
26 63
81 49
55 27
56 45
66 50
14 62
39 74
75 17
73 45
48 3
17 84
61 11
75 41
1 33
78 12
9 12
45 46
20 48
41 23
58 22
54 62
37 3
37 63
29 38
46 31
66 40
76 57
78 66
