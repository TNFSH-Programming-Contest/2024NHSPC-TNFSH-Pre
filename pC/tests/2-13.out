50
1 3 5 10 11 12 13 14 15 18 20 21 22 23 25 27 28 29 32 33 34 35 36 40 41 42 44 45 51 54 55 56 57 58 59 60 61 65 68 69 70 71 72 73 75 78 80 81 82 83 